### PR TITLE
Added a default github_token

### PR DESCRIPTION
**Pull request by jackton1**
@ad-m

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -6,8 +6,9 @@ branding:
   color: green
 inputs:
   github_token:
-    description: 'Token for the repo. Can be passed in using $\{{ secrets.GITHUB_TOKEN }}'
+    description: 'Github token or PAT token'
```
Comment (ad-m) [inline action.yml:9]:
```suggestion
    description: 'GitHub token or PAT token'
```

**Comment by osdiab** [comment]
Looks like this broke the script since the syntax is invalid, my company’s builds started failing @ad-m

File: action.yml [hunk 2]
```diff
@@ -6,8 +6,9 @@ branding:
   color: green
 inputs:
   github_token:
-    description: 'Token for the repo. Can be passed in using $\{{ secrets.GITHUB_TOKEN }}'
+    description: 'GitHub token or PAT token'
     required: true
+    default: ${{ github.token }]
```
Comment (osdiab) [inline action.yml:11]: This is the faulty line

**Comment by ad-m** [comment]
@osdiab , hot fix published

**Pull request merged**